src/main/java/uk/gov/ons/census/fwmtadapter/messaging/ActionFieldReceiver.java

```diff
@@ -54,6 +54,10 @@ public void receiveMessage(FieldworkFollowup followup) {
     actionInstruction.setUprn(followup.getUprn());
     actionInstruction.setUndeliveredAsAddress(followup.getUndeliveredAsAddress()); // TODO: Delete?
 
+    if (followup.getAddressType().equals("CE")) {
```

Comment:
Would prefer not to have this CE check, I presume we're ok sending SecureEstab == False to field for none CE cases?  I thought this might affect SPGs too?

Reply:
We're going to raise another card for SPGs